modules/imaging_qc/jsx/imagingQCIndex.js

```diff
@@ -253,10 +254,10 @@ ImagingQCIndex.propTypes = {
 };
 
 window.addEventListener('load', () => {
-  createRoot(
-    document.getElementById('lorisworkspace')
-  ).render(
-    <ImagingQCIndex
+  i18n.addResourceBundle('jp', 'loris', {}, true, true);
```

Comment:
I don't think you need to call addResourceBundle, and if you do you're overwriting the loris namespace here with nothing.